docs/includes/enable-debugging-on-edge-devtools.md

```diff
@@ -0,0 +1,38 @@
+When the add-in is running in Microsoft Edge, UI-less code will not be able to attach to a debugger by default.
+UI-less code is any code running while the task pane is not visible, such as add-in commands. To enable debugging, you need to run the following [Windows PowerShell](https://docs.microsoft.com/powershell/scripting/getting-started/getting-started-with-windows-powershell) commands.
+
+1. Run the following command to get information for the **Microsoft.Win32WebViewHost** app package.
```

Comment:
This is ok as is, but the following is preferred:
1. To get information for the **Microsoft.Win32WebViewHost** app package, run the following command.